fix(compnents): remove the usage of stencils build conditionals in utility modules (#7559)
swisspost/design-system: apps/documentation/src/stories/packages/components-react/components-react.docs.mdx

<Source code={GlobalCSSSample} language="scss" />

> The package provides different entry points for CSR (Client-Side Rendering) and SSR (Server-Side Rendering). Each is available as a **monolith** import (all components in one bundle) or as **standalone** imports (individual components for better tree-shaking).

Reviewer: ```suggestion
The package provides different entry points for CSR (Client-Side Rendering) and SSR (Server-Side Rendering). Each is available either as a **monolithic** import (all components in one bundle) or as **standalone** imports (individual components for better tree-shaking).
```

### Client side rendering (CSR)

> If you're project is purely client-side rendered, you can use the CSR entry points.

Reviewer: ```suggestion
If your project is purely client-side rendered, you can use the CSR entry points.
```

<post-tabs>
  <post-tab-item name="csr-monolith">Monolith</post-tab-item>
  <post-tab-panel for="csr-monolith">
    The monolith entry point is the simplest way to get started.
    <Source code={CSRSample} language="typescript" />
  </post-tab-panel>

  <post-tab-item name="csr-standalone">Standalone</post-tab-item>
  <post-tab-panel for="csr-standalone">

>     The standalone entry point gives you fine grained control over your imports.

Reviewer: ```suggestion
    The standalone entry point gives you fine-grained control over your imports.
```

    <Source code={CSRStandaloneSample} language="typescript" />
  </post-tab-panel>
</post-tabs>

### Server side rendering (SSR)

The SSR entry points are needed for server-side rendering.

<post-banner variant="info">

> Using these entry points instead of the above ones, is highly recommended for Next.js projects!

Reviewer: ```suggestion
Using these entry points instead of the above ones is highly recommended for Next.js projects!
```

</post-banner>

<post-tabs>
  <post-tab-item name="ssr-monolith">Monolith</post-tab-item>
  <post-tab-panel for="ssr-monolith">
    The monolith entry point is the simplest way to get started.
    <Source code={SSRSample} language="typescript" />
  </post-tab-panel>

  <post-tab-item name="ssr-standalone">Standalone</post-tab-item>
  <post-tab-panel for="ssr-standalone">
    The standalone entry point gives you fine grained control over your imports.
    <Source code={SSRStandaloneSample} language="typescript" />
  </post-tab-panel>
</post-tabs>

### Icons

The package includes generated React icon components, built from `@swisspost/design-system-icons`,
therefore it's no longer necessary to have the icons package installed as a dependency to use icons.
Under the hood, the `<post-icon>` component is being used to render the icons consistently,

> therefore also all the properties from this component are available for the react icons.

Reviewer: ```suggestion
therefore, all properties of this component are also available for the React icons.
```

> See the [icon documentation](https://next.design-system.post.ch/?path=/docs/0dcfe3c0-bfc0-4107-b43b-7e9d825b805f--docs) for more information on it's features.

Reviewer: ```suggestion
See the [icon documentation](https://next.design-system.post.ch/?path=/docs/0dcfe3c0-bfc0-4107-b43b-7e9d825b805f--docs) for more information on its features.
```

<post-banner variant="info">

> The icon components are currently only available with integrated SSR compatibility. However, they can be used on pure client-side projects as well.

Reviewer: ```suggestion
The icon components are currently only available with built-in SSR compatibility. However, they can be used on purely client-side projects as well.
```
